src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -272,6 +282,7 @@
         (ElectraConfig, TFElectraForPreTraining),
         (FunnelConfig, TFFunnelForPreTraining),
         (MPNetConfig, TFMPNetForMaskedLM),
+        (LayoutLMConfig, TFLayoutLMForMaskedLM),
```

Comment:
```suggestion
```
It's already defined up there.

Reply:
fixed [here](https://github.com/huggingface/transformers/pull/10636/commits/3bee70daf2a71066335d360761ce5c7bb432500a)